Core/tests/Unit/RequestWrapperTest.php

```diff
@@ -625,6 +625,29 @@ public function testEmptyTokenThrowsException()
 
         $requestWrapper->send(new Request('GET', 'http://www.example.com'));
     }
+
+    public function testRetryOptionsPassingInGetRetryOptions()
+    {
+        $requestWrapper = new RequestWrapper();
+        $reflection = new \ReflectionClass($requestWrapper);
+        $reflectionMethod = $reflection->getMethod('getRetryOptions');
+        $reflectionMethod->setAccessible(true);
+
+        $placeholderCallback = function () {
+        };
+        $options = [
+            'restRetryFunction' => $placeholderCallback,
+            'restOnRetryExceptionFunction' => $placeholderCallback,
+            'restOnExecutionStartFunction' => $placeholderCallback,
+        ];
+
+        $result = $reflectionMethod->invoke($requestWrapper, $options);
```

Comment:
This part isn't intuitive, what's happening here?

Reply:
Updated docbloc. [9e155ff](https://github.com/googleapis/google-cloud-php/pull/5953/commits/9e155ff90545e3b7a65efa01492261db4cdc874e)

Reply:
Small comment above the `foreach` will also go a long way :)

Reply:
Now that I see it, this was neccesary to comment! Thanks for calling this out.

Updated in commit [ba03c51](https://github.com/googleapis/google-cloud-php/pull/5953/commits/ba03c51d90bb70cd400d755623229350d97ccd2d)